### DISCUSSION TITLE
Virtual Swag

### DISCUSSION THREAD

**Discussion by AndreaGriffiths11** [gittogethers/octogatosconf22#8] [General]
Ideas

**Comment by AndreaGriffiths11** [comment]
<img width="539" alt="Screen Shot 2022-06-28 at 12 11 40 PM" src="https://user-images.githubusercontent.com/20666190/176228663-48ff98d3-b1db-46a8-89a0-77c47c20f4d8.png">

**Reply by brunobarros2093** [reply]
Some virtual rooms like gather town would be fun too

**Comment by AndreaGriffiths11** [comment]
[vercel app maybe for next year](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fvercel%2Fvirtual-event-starter-kit&project-name=virtual-event-starter-kit&repository-name=virtual-event-starter-kit&demo-title=Virtual%20Event%20Starter%20Kit&demo-description=Jumpstart%20your%20virtual%20event%20and%20scale%20to%20any%20size%20with%20Next.js%20and%20Vercel.&demo-url=https%3A%2F%2Fdemo.vercel.events%2F&demo-image=https%3A%2F%2Fdemo.vercel.events%2Fdeploy.png&integration-ids=oac_7yeSwUoVR5no3SlA9WM6oR7l)

**Comment by lizarragadev** [comment]
I can think of creating a **filter** for _Instagram stories_, something like: What Octocat are you? and randomly a random Octocat comes out, I read that it can be done with _SparkAR_, I don't know much about it, but I could investigate.
Doing this would help each participant identify with OctogatosConf and generate publicity for us.

An example would be as seen in the image.

![maxresdefault](https://user-images.githubusercontent.com/1924390/176339644-b88beacf-ad5b-4b8c-ba9d-0a83db898128.jpeg)

**Reply by AndreaGriffiths11** [reply]
This is super fun, yes please @lizarragadev

**Reply by lizarragadev** [reply]
Sure! For next week I will had a demo! 😄

**Reply by pachicodes** [reply]
This is such a a fun idea!

**Reply by lizarragadev** [reply]
This is a small demo, improving the images and other but something initial 😄 

https://user-images.githubusercontent.com/1924390/179029331-9feda4ae-f984-4f52-a988-9f7fc8e31958.mov

**Reply by pachicodes** [reply]
awesome!!!

**Reply by lizarragadev** [reply]
@AndreaGriffiths11  Acá arriba esta el demo :)

**Reply by AndreaGriffiths11** [reply]
LOVE!!! How do we get this published? Will it work in TikTok? I'm doing a short to promote the conference this weekend and it'll be cool to use it

**Reply by lizarragadev** [reply]
@AndreaGriffiths11  I'm not sure if it works on TikTok, but I'll investigate how to upload it and make it available on boths platforms, Instagram and TikTok.

**Comment by pachicodes** [comment]
I think we should have a certified as well, even if it is "boring". In Brazil students can use that to count towards credit in College

**Reply by lizarragadev** [reply]
Yes, in Bolivia it also catches their attention and in some cases the certificates are useful :)

**Reply by AndreaGriffiths11** [reply]
Do either or you know how to go about getting a certification?

**Reply by lizarragadev** [reply]
@AndreaGriffiths11  about the technical part or how generate certificates, There is a tool in Python that is to generate certificates in base to CSV file, the tool is PILLOW, only execute a script and that's it! 😬

**Reply by AndreaGriffiths11** [reply]
Wonderful! I think we should do a intro to Git & GitHub session in each language and that be the verifiable certification. @pachicodes wiuld you be up for leading this? 20- 30 minutes, I'll teach the one in Spanish and maybe one of our non Spanish speaking Octogatos can do one in English.

**Reply by pachicodes** [reply]
@AndreaGriffiths11  Yes!!!

**Comment by pachicodes** [comment]
Maybe we could make a "frame" for Twitter and LinkedIn profile pictures

**Comment by pachicodes** [comment]
Hi friends,
I went to a conference this weekend and they had a super cool online swag.
You would send a picture of you enjoying the event to Twilio's whatsapp, and after a few minutes you would get [a customize video like this one](https://drive.google.com/file/d/1lZcLsAxmRJTC2kCzLYXBq0AxCH7Wg4gN/view?usp=sharing) 

It was made in partnership with [Videomatik](https://videomatik.com.br/), an API to produce personalized videos in large scale. 
I talked with someone from this company, and they were super nice.

👀 @AndreaGriffiths11

**Comment by AndreaGriffiths11** [comment]
Love THAT wonder how much videomatik charges?

**Reply by pachicodes** [reply]
I will schedule a call with them to ask that and understand better how they can help us (do they offer the art template, or do we, can we have one template for each language? etc)
So let me know if you have any question you would like me to ask them

**Comment by AndreaGriffiths11** [comment]
![image](https://user-images.githubusercontent.com/20666190/180607310-3616fbdf-5445-4fda-8b65-7f8b3b2eec6b.png)

![image](https://user-images.githubusercontent.com/20666190/180607320-c8af5384-a525-426c-b333-5750518919c0.png)

**Comment by shyanukant** [comment]
**NFT is the best idea for [virtual swag](https://www.fastcompany.com/90674430/these-startups-are-creating-nfts-and-digital-swag-for-the-metaverse-middle-class) at this time because everyone is excited about web3.0 and blockchain**
You can gift `3D NFTs` as swag.

[Checkout alchemy](https://docs.alchemy.com/docs/welcome-to-the-road-to-web3)

**Comment by Hrushal-Nikhare** [comment]
maybe like a globe map which shows the number of people from a county on a 3d globe
